graphql/data/schema.js

```diff
@@ -1679,7 +1687,7 @@ const updateImpactMutation = mutationWithClientMutationId({
   name: 'UpdateImpact',
   inputFields: {
     userId: { type: new GraphQLNonNull(GraphQLString) },
-    charityId: { type: new GraphQLNonNull(GraphQLString) },
+    charityId: { type: GraphQLString },
```

Comment:
beginning deprecating sending up charity ID since we now get it from a user's cause